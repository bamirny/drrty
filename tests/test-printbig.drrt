
def int main(){
  int i;
  int j;
  createHTMLDocument("printbig.css");
  for (i = 0; i < 100; i = i + 1){
      if( j == 0){
        createElement("div", "bigClass", ""); 
        j = 1; 
      }else{
        createElement("div", "smallClass", "");  
        j = 0;
      }
  }
  return 0;
}



  