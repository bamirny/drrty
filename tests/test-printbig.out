<!DOCTYPE html><html><head><link rel="stylesheet" href="printbig.css"></head><body><div class="smallClass"></div>
<div class="bigClass"></div>
<div class="smallClass"></div>
<div class="bigClass"></div>
<div class="smallClass"></div>
<div class="bigClass"></div>
<div class="smallClass"></div>
<div class="bigClass"></div>
<div class="smallClass"></div>
<div class="bigClass"></div>
<div class="smallClass"></div>
<div class="bigClass"></div>
<div class="smallClass"></div>
<div class="bigClass"></div>
<div class="smallClass"></div>
<div class="bigClass"></div>
<div class="smallClass"></div>
<div class="bigClass"></div>
<div class="smallClass"></div>
<div class="bigClass"></div>
<div class="smallClass"></div>
<div class="bigClass"></div>
<div class="smallClass"></div>
<div class="bigClass"></div>
<div class="smallClass"></div>
<div class="bigClass"></div>
<div class="smallClass"></div>
<div class="bigClass"></div>
<div class="smallClass"></div>
<div class="bigClass"></div>
<div class="smallClass"></div>
<div class="bigClass"></div>
<div class="smallClass"></div>
<div class="bigClass"></div>
<div class="smallClass"></div>
<div class="bigClass"></div>
<div class="smallClass"></div>
<div class="bigClass"></div>
<div class="smallClass"></div>
<div class="bigClass"></div>
<div class="smallClass"></div>
<div class="bigClass"></div>
<div class="smallClass"></div>
<div class="bigClass"></div>
<div class="smallClass"></div>
<div class="bigClass"></div>
<div class="smallClass"></div>
<div class="bigClass"></div>
<div class="smallClass"></div>
<div class="bigClass"></div>
<div class="smallClass"></div>
<div class="bigClass"></div>
<div class="smallClass"></div>
<div class="bigClass"></div>
<div class="smallClass"></div>
<div class="bigClass"></div>
<div class="smallClass"></div>
<div class="bigClass"></div>
<div class="smallClass"></div>
<div class="bigClass"></div>
<div class="smallClass"></div>
<div class="bigClass"></div>
<div class="smallClass"></div>
<div class="bigClass"></div>
<div class="smallClass"></div>
<div class="bigClass"></div>
<div class="smallClass"></div>
<div class="bigClass"></div>
<div class="smallClass"></div>
<div class="bigClass"></div>
<div class="smallClass"></div>
<div class="bigClass"></div>
<div class="smallClass"></div>
<div class="bigClass"></div>
<div class="smallClass"></div>
<div class="bigClass"></div>
<div class="smallClass"></div>
<div class="bigClass"></div>
<div class="smallClass"></div>
<div class="bigClass"></div>
<div class="smallClass"></div>
<div class="bigClass"></div>
<div class="smallClass"></div>
<div class="bigClass"></div>
<div class="smallClass"></div>
<div class="bigClass"></div>
<div class="smallClass"></div>
<div class="bigClass"></div>
<div class="smallClass"></div>
<div class="bigClass"></div>
<div class="smallClass"></div>
<div class="bigClass"></div>
<div class="smallClass"></div>
<div class="bigClass"></div>
<div class="smallClass"></div>
<div class="bigClass"></div>
<div class="smallClass"></div>
<div class="bigClass"></div>
<div class="smallClass"></div>
<div class="bigClass"></div>
